tank_teleop_node:
  ros__parameters:
    axis_left: 1   # Left stick vertical
    axis_right: 4  # Right stick vertical
    scale_linear: 0.5
    scale_angular: 1.0
    enable_button: 10 # L1
